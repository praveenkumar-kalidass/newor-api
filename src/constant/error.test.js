const error = require('./error');

describe('error', () => {
  it('should match snapshot', () => {
    expect(error).toMatchSnapshot();
  });
});
