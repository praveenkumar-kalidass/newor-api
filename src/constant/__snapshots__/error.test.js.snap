// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error should match snapshot 1`] = `
Object {
  "BAD_REQUEST": Object {
    "data": Object {
      "code": "NEWOR_BAD_REQUEST",
      "description": "Bad request",
    },
    "status": 400,
  },
  "INTERNAL_SERVER_ERROR": Object {
    "data": Object {
      "code": "NEWOR_INTERNAL_SERVER_ERROR",
      "description": "Internal Server error",
    },
    "status": 500,
  },
  "REDIRECT": Object {
    "status": 307,
  },
  "UNAUTHORIZED": Object {
    "data": Object {
      "code": "NEWOR_UNAUTHORIZED",
      "description": "Unauthorized",
    },
    "status": 401,
  },
}
`;
